From: Marco Felsch <m.felsch@pengutronix.de>
Date: Tue, 17 Jun 2025 13:14:20 +0200
Subject: [PATCH] drivers: imx: tzc380: add CFG_MX6QP TZASC2 configuration

The i.MX6DP/QP SoCs have a 2nd memory controller as well which must be
configured.

This commit covers only the i.MX6QP because there is no i.MX6DP OP-TEE
plattform yet.

Signed-off-by: Marco Felsch <m.felsch@pengutronix.de>
---
 core/arch/arm/plat-imx/tzc380.c | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/core/arch/arm/plat-imx/tzc380.c b/core/arch/arm/plat-imx/tzc380.c
index ecc545be6948..2350221d7a77 100644
--- a/core/arch/arm/plat-imx/tzc380.c
+++ b/core/arch/arm/plat-imx/tzc380.c
@@ -56,7 +56,7 @@ static TEE_Result imx_configure_tzasc(void)
 	addr[0] = core_mmu_get_va(TZASC_BASE, MEM_AREA_IO_SEC, 1);
 
 	if (IS_ENABLED(CFG_MX6Q) || IS_ENABLED(CFG_MX6D) ||
-	    IS_ENABLED(CFG_MX6DL)) {
+	    IS_ENABLED(CFG_MX6DL) || IS_ENABLED(CFG_MX6QP)) {
 		assert(TZASC2_BASE != 0);
 		addr[1] = core_mmu_get_va(TZASC2_BASE, MEM_AREA_IO_SEC, 1);
 		end = 2;
